http/action/src/main/java/io/knotx/databridge/http/action/HttpAction.java

```diff
@@ -232,7 +260,22 @@ private ActionPayload handleInvalidResponseBodyFormat(ActionRequest request, Dec
   }
 
   private ActionPayload handleSuccessResponse(EndpointResponse response, ActionRequest request) {
-    return ActionPayload.success(request, bodyToJson(response.getBody().toString()));
+    if (httpActionOptions.getResponseOptions().isForceJson() ||
```

Comment:
The `forceJson` is created to keep backward compatibility? Why is it needed?
If we set `forceJson = true` and body of a response is not a JSON we get the exception right?

Reply:
Yes, you're right, the behavior of forceJson and whole httpAction is described in more detail in the documentation which was added in another pull request (https://github.com/Knotx/knotx-data-bridge/pull/58), there we also provided a table with the expected behavior of this action depending on provided config and http response

Reply:
it's in another pull request because when I've started working on this fork, I didn't have rights to push directly to Knot.x repository, some changes were pushed there and this branch was also merged with proper one in the main repository